### DISCUSSION TITLE
No Option For Disabling Floating URL Bar On Search As Mentioned In Release Notes?

### DISCUSSION THREAD

**Discussion by Muhammad-Mohid-Khan** [zen-browser/desktop#3626] [General]
cant find it anywhere can someone tell me where it is???

**Comment by LaserKaspar** [comment]
It can be "reverted" using preferences:
https://github.com/zen-browser/desktop/issues/3649

But then it is transparent, so not really usable for me sadly...

Then new bar does not really seem like it was tested properly. Also the experience it horrible when trying to change something in the urlbar. Click top, move mouse to center, click where yo wanted to click in the url again. Instead of just being able to click... 

Opening a new tab does not open the searchbar. So it also misses the thing that was good about the floating bar in Arc...

**Reply by Sinaaaa** [reply]
> It can be "reverted" using preferences: #3649
> 
> But then it is transparent, so not really usable for me sadly...
> 
> Then new bar does not really seem like it was tested properly. Also the experience it horrible when trying to change something in the urlbar. Click top, move mouse to center, click where yo wanted to click in the url again. Instead of just being able to click...
> 
> Opening a new tab does not open the searchbar. So it also misses the thing that was good about the floating bar in Arc...

You can make it opaque with userChrome.css

```
@media (-moz-bool-pref: "zen.view.use-deprecated-urlbar") {
  #urlbar[breakout-extend="true"] #urlbar-background {
    background-color: #1e202f !important;
  }
}
```

But yes, I think the new floating URL bar is not great & even if you fix it with doing these things there are still some url bar bugs that were not present before "beta".
